esmvalcore/_provenance.py

```diff
@@ -173,14 +193,18 @@ def _initialize_entity(self):
         }
         self.entity = self.provenance.entity('file:' + self.filename,
                                              attributes)
+
         attribute_to_authors(self.entity, self.attributes.get('authors', []))
         attribute_to_projects(self.entity, self.attributes.get('projects', []))
 
     def _initialize_ancestors(self, activity):
         """Register ancestor files for provenance tracking."""
         for ancestor in self._ancestors:
             if ancestor.provenance is None:
-                ancestor.initialize_provenance(activity)
+                if os.path.exists(ancestor.provenance_file):
+                    ancestor.restore_provenance()
```

Comment:
hoping this will not throw a KeyError with the usual "prov file exists blah..."

Reply:
I'm not too worried about that :beers: